contrib/docker/dockerfile

```diff
@@ -0,0 +1,38 @@
+# build stage
+## import image
+FROM golang:alpine AS build_base
+
+## add git
+RUN apk add --no-cache git
+
+## cd into global 700 dir (image specific)
+WORKDIR /go
+
+## populate the module cache based on the go.{mod,sum} files.
+COPY go.mod .
+COPY go.sum .
+## download the modules
+RUN go mod download
+
+## copy all files into build stage
+COPY . .
```

Comment:
Why did you explicitly `COPY`ed `go.{mod,sum}` before, and now `COPY` everything?